release/nightly_tests/dataset/image_loader_microbenchmark.py

```diff
@@ -128,9 +196,27 @@ def get_transform(to_torch_tensor):
     )
     return transform
 
+transform = torchvision.transforms.Compose(
```

Comment:
looks like this `transform` gets overwritten when we call `get_transform` below? should we remove it? (possibly leftover from debugging?)